packages/apps/crm/src/types/event-type.ts

```diff
@@ -56,6 +56,20 @@ class EventType {
     }
   };
 
+  static getCategoryList = (I18n: {t: (key: string) => string}) => {
+    return Object.entries(this.category).map(([key, value]) => ({
+      title: I18n.t(`Crm_Event_Category_${key}`),
+      key: value,
+    }));
+  };
+
+  static getStatusList = (I18n: {t: (key: string) => string}) => {
+    return Object.entries(this.status).map(([key, value]) => ({
+      title: I18n.t(`Crm_Event_Status_${key}`),
+      key: value,
+    }));
+  };
```

Comment:
Great to have it in this file, but can you organize the file to keep it readable ? 
* getStatus
* getStatusBorderColor
* getStatusList
* getCategory
* getCategoryList

It's easier to find a function this way.
Also I think you can add the color to the statusList